api/views.py

```diff
@@ -131,19 +123,16 @@ def stand(request, id):
     # load deck from GameState into new game instance
     blackjack_game.deck_cards = db_deck
 
-    # considerign making this into an inner function?
     # deal card and add to dealer's hand
     card_dealt = blackjack_game.deal()
     dealer_hand.append(card_dealt)
-    print(dealer_hand)
-    # I don't think i need to do this here with only one card
-    db_game.dealer_hand = json.dumps(dealer_hand)
+
     db_player_hand = json.loads(db_game.player_hand)
 
     db_game.save()
 
     dealer_score = blackjack_game.calculate_cards(dealer_hand)
-    print(dealer_score)
+
     while True:
         if dealer_score == 21:
             db_game.active = False
```

Comment:
instead of setting db_game active to False, could we do it at the end of this function?

Reply:
I had to think through this for a minute, but I see it now. Once it breaks, it would be false regardless of the scenario, so yes.